org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/data/DataEntryField.java

```diff
@@ -150,4 +154,31 @@ public static Integer getPropertyAsInt(Object data, String fieldName) {
 		return element != null ? element.getAsInt() : null;
 	}
 
+	/**
+	 * Returns the value of the field as an boolean or null if the field is empty.
+	 *
+	 * @param data      the object to read the field of
+	 * @param fieldName the field to read
+	 * @throws ClassCastException if the field is not an boolean
+	 * @return the value of the field as an boolean or null if the field is empty
+	 */
+	public static Boolean getPropertyAsBoolean(Object data, String fieldName) {
+		// FIXME: catch exception return null

```

Comment:
Address the `FIXME:` comment

Reply:
There is this FIXME because I copied / pasted from getPropertyAsInt, but I don't know the problem. Could we let like this like existing getPropertyAsInt?

Reply:
Okay. Thinking back to it, this shouldn't be a huge problem, since this is data that we generate, so it should always have the right type.